native/src/routes/CityNotCooperating.tsx

```diff
@@ -0,0 +1,128 @@
+import Clipboard from '@react-native-clipboard/clipboard'
+import React, { ReactElement, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Button } from 'react-native-elements'
+import { useTheme } from 'styled-components'
+import styled from 'styled-components/native'
+
+import CityIcon from '../assets/cityNotCooperating.svg'
+import { template } from '../constants/cityNotCooperatingTemplate'
+
+const Container = styled.ScrollView`
+  display: flex;
+  padding: 30px;
+`
+
+const HeadingContainer = styled.View`
+  max-width: 70%;
+  align-self: center;
+`
+
+const Heading = styled.Text`
+  font-family: ${props => props.theme.fonts.native.decorativeFontBold};
+  font-size: 18px;
+  padding: 20px;
+  text-align: center;
+`
+
+const Description = styled.Text`
+  font-family: ${props => props.theme.fonts.native.contentFontRegular};
+`
+
+const ListHeading = styled(Heading)`
+  text-align: left;
+  font-size: 15px;
+  padding: 10px 0;
+`
+
+const ListItem = styled.View`
+  flex-direction: row;
+  margin: 10px 0;
+  align-items: center;
+`
+
+const StepNumber = styled.Text`
+  background-color: ${props => props.theme.colors.themeColor};
+  font-size: 15px;
+  line-height: 28px;
+  text-align: center;
+  width: 30px;
+  height: 30px;
+  border-radius: 15px;
+  margin-right: 10px;
+`
+
+const StepExplanation = styled.Text`
+  align-self: center;
+  flex-shrink: 1;
+  padding: 0 0 4px 0;
+`
+
+const Icon = styled.Image`
+  align-self: center;
+  width: 50%;
+  resize-mode: contain;
+`
+
+const ButtonContainer = styled.View`
+  z-index: 1;
```

Comment:
Do we really need a z-index here?

Reply:
yes otherwise the border of the View below with the template text will be above the Button-Text